packages/share/lib/share.dart

```diff
@@ -3,6 +3,7 @@
 // found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:io';
```

Comment:
Thinking of a future version for web here. Do we need to share `dart:io` File objects? Can we tell the native code to share "file paths" or similar, and have the file be created in the native side?

Depending on `dart:io` File objects guarantees that we'll have to rewrite this if we want to add Web support (even though [the API](https://caniuse.com/#feat=web-share) is not very well supported yet)

Reply:
Makes sense. I have quickly changed it and confirmed it still works. Only difference now is that if a non existent path is passed in, the native side will throw an error. But I guess that is okay and the expected result. Before it threw an assertion error from the dart side.